csm_web/scheduler/serializers.py

```diff
@@ -27,7 +27,10 @@ class Meta:
 class AttendanceSerializer(serializers.ModelSerializer):
     class Meta:
         model = Attendance
-        fields = ("id", "presence", "week_start")
+        fields = ("id", "presence", "week_start", "date", "student")
+        read_only_fields = ("week_start",)
+        extra_kwargs = {'student': {'write_only': True},
```

Comment:
Isn't is annoying that DRF doesn't have a `write_only_fields` for its `Meta` classes?

Reply:
yeah that's pretty annoying